doc/manuals/language.am

```diff
@@ -6,16 +6,18 @@
 ## Please see the COPYING and CONTRIBUTORS files for details.
 ##
 
+## This file is generated by scripts/source-maintenance.sh.
+
 TRANSLATE_LANGUAGES = \
-    ar.lang \
-    cs.lang \
-    de.lang \
-    en_AU.lang \
-    en.lang \
-    es.lang \
-    fr.lang \
-    it.lang \
-    oc.lang \
-    pt.lang \
-    ro.lang \
-    ru.lang
+	ar.lang \
+	cs.lang \
+	de.lang \
+	en.lang \
+	en_AU.lang \
+	es.lang \
```

Comment:
Lots of whitespace changes in Makefiles because generated files were not using tabs. To see essential changes on GitHub, tell GitHub to hide whitespace changes.
![Screenshot from 2022-07-03 23-43-37](https://user-images.githubusercontent.com/1084944/177077960-172c745b-29f7-4627-bf02-761b19d04357.png)